package.json

```diff
@@ -6,15 +6,15 @@
   },
   "scripts": {
     "test": "jest",
-    "test:full": "npm run build && npm run test:coverage",
+    "test:full": "npm run test:coverage",
     "test:flow": "flow check --all --show-all-errors",
     "test:unit": "jest",
     "test:coverage": "jest --coverage",
     "test:watch": "npm test -- --watch",
     "test:unit-watch": "jest --watch",
     "test:coveralls": "cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js && rm -rf ./coverage",
     "start": "webpack-dev-server --open --config webpack.dev.js",
-    "clean": "rimraf dist",
+    "clean": "rm -rf dist",
```

Comment:
@brucebinary This is not going to work for Windows users

Reply:
owh. I forgot to mention why I changed that. The rimraf thing doesn't work in travis once I changed to the android env - not sure why. Let me take another look at that

Are we suppose to support windows?

Reply:
This is a open source project. Sometimes 3rd parties fork this project and do their own thing. I also remember, one of the telecommuters was also using Windows for development. Its an on/off thing. I think @nazaninreihani might also be using Windows(not sure). So its better to support windows when possible. That `rimraf` probably does not work in windows either.

Reply:
@arnabk @brucebinary I'm using ubuntu, no issue for ubuntu I think, yes?

Reply:
travis runs on ubuntu, so I don't think so